ko-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -37,7 +37,9 @@
         <presenters.version>${presenters.version}</presenters.version>
         <enforcer.fail>false</enforcer.fail>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <nashorn.version>${nashorn.version}</nashorn.version>
+        <!--<nashorn.version>${nashorn.version}</nashorn.version>-->
+        <graaljs.version>${graaljs.version}</graaljs.version>
```

Comment:
Finding a version of Graal.js that works on JDK21 and JDK25 should be possible, but I am not exactly sure what's OracleLabs compatibility policy for the future (CCing @chumer). Will there be a version that can (somehow, not necessarily at full speed) run on JDK21 for next ten (let's say) years?

If not, then it might be easier to stick with Nashorn...